components/tidb_query/src/codec/row/v2.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{i16, i32, i8, u16, u32, u8};
+
+pub const CODEC_VERSION: u8 = 128;
+const BIG_ID_FLAG: u8 = 1;
+
+mod row_slice;
+
+#[cfg(test)]
+mod encoder;
+
+trait BoundedU64 {
```

Comment:
maybe this is what you want: https://rust-num.github.io/num/num/trait.Bounded.html XD